kubemarine/procedures/check_iaas.py

```diff
@@ -699,15 +699,17 @@ def get_ports_connectivity(cluster: KubernetesCluster, proto: str) -> Dict[str,
         }
 
     else:  # udp
+        overlay = cluster.inventory['plugins']['calico']['mode']
+        vxlan_port = ['4789'] if overlay == 'vxlan' else []
         connectivity_ports = {
             'internal': {
                 'input': {
-                    'control-plane': ['53'],
-                    'worker': ['53']
+                    'control-plane': ['53'] + vxlan_port,
+                    'worker': ['53'] + vxlan_port
```

Comment:
This fails on the installed cluster.

Reply:
It's expected, verbose log has the following messages:
```
2023-12-17 19:24:55,496 139864351342656 VERBOSE [log.verbose] UDP port 4789 is already in use on node-3
2023-12-17 19:24:55,496 139864351342656 VERBOSE [log.verbose] UDP port 4789 is already in use on node-2
2023-12-17 19:24:55,497 139864351342656 VERBOSE [log.verbose] UDP port 4789 is already in use on node-1
```

It could be checked by the following command on some of the node:
```
# netstat -nplA inet |grep '4789'
udp        0      0 0.0.0.0:4789            0.0.0.0:*                
```

Reply:
In this case we need to change the behavior for all `ports_connectivity` check

Reply:
I do not mind as long as the new behaviour will impact only this new port.